client/js/lounge.js

```diff
@@ -1,3 +1,15 @@
+import $ from "jquery";
+import "jquery-ui/ui/widgets/sortable";
+import Mousetrap from "mousetrap";
+import Handlebars from "handlebars/runtime";
+import "notification-polyfill";
+
+import "./libs/jquery/inputhistory";
+import "./libs/jquery/tabcomplete";
+import "./libs/jquery/stickyscroll";
```

Comment:
@astorije These "libraries" were so heavily modified that I would consider them part of Lounge's code and not vendor.